ca_gazebo/launch/create_apriltag.launch

```diff
@@ -0,0 +1,27 @@
+<launch>
+	<arg name="env" default="apriltags"/>
+  	<arg name="paused" value="$(optenv PAUSED false)"/>
+	<arg name="gui" value="$(optenv GUI true)"/>
+	<arg name="debug" value="$(optenv DEBUG false)"/>
+	<env name="GAZEBO_MODEL_PATH" value="$(find ca_gazebo)/models"/>
+
+	<!-- Launch Gazebo world -->
+	<include file="$(find gazebo_ros)/launch/empty_world.launch">
+		<arg name="debug" value="$(arg debug)" />
+		<arg name="gui" value="$(arg gui)" />
+		<arg name="paused" value="$(arg paused)"/>
+		<arg name="use_sim_time" value="true"/>
+		<arg name="verbose" value="true"/>
+		<arg name="world_name" value="$(find ca_gazebo)/worlds/$(arg env).world"/>
+	</include>
+	<node pkg="tf" type="static_transform_publisher" name="camera_link_broadcaster" args="2.486 -1.417 9.80 3.14159 3.14159 0 map april_camera_link 100" />
+
+	<!-- Spawn robot/s -->
```

Comment:
I don't know if I already told you this, but why are these calls here?

Reply:
Why don't you call [this](https://github.com/RoboticaUtnFrba/create_autonomy/blob/kinetic-devel/ca_gazebo/launch/create_house.launch#L10-L12) instead?